t/Equaldex.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::Equaldex)],
+
+    'lgbt china' => test_spice(
+        '/js/spice/equaldex/china',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+
+    'lesbian rights cyprus' => test_spice(
+        '/js/spice/equaldex/cyprus',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'gay rights australia' => test_spice(
+        '/js/spice/equaldex/australia',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'transgender rights usa' => test_spice(
```

Comment:
@MrChrisW thanks for explaining. I understand now. It seems like we need a way to ignore/define our own aliases which should be do-able.

/cc @zachthompson 

In the interim I would suggest you use the `all_countries()` array -- I assume that doesn't contain `us` and `usa`? If not, then you can make a hash out of the country-names array and do a lookup to see if the country exists.

---

I stil don't get why `country2code("usa")` doesn't return "US"; If it did, couldn’t you' just get the correct country name by reversing that? I think that would let you support queries like "LGBT rights usa", right?